src/pudl/workspace/datastore.py

```diff
@@ -206,7 +206,7 @@ class ZenodoDoiSettings(BaseSettings):
     ferc60: ZenodoDoi = "10.5281/zenodo.13149090"
     ferc714: ZenodoDoi = "10.5281/zenodo.13149091"
     gridpathratoolkit: ZenodoDoi = "10.5281/zenodo.10892394"
-    phmsagas: ZenodoDoi = "10.5281/zenodo.10493790"
+    phmsagas: ZenodoDoi = "10.5281/zenodo.13624526"
```

Comment:
@seeess1 One more thing to flag - we published new PHMSA data this month, so the DOI should now be:

```suggestion
    phmsagas: ZenodoDoi = "10.5281/zenodo.14026420"
```

Reply:
Updated